src/components/clusterWizard/ClusterDetails.tsx

```diff
@@ -252,11 +254,21 @@ const ClusterDetailsForm: React.FC<ClusterDetailsFormProps> = (props) => {
                 <Alerts />
               </StackItem>
             )}
+            {isValidationSectionOpen && cluster && (
```

Comment:
This is repeating on every step. Let's keep it in `ClusterWizardToolbar`. We could move whole `<Stack>` into a `ClusterWizardToolbar` component. Including Alerts.

Reply:
good point, moved